backend/fms_core/services/experiment_run_info.py

```diff
@@ -245,6 +248,16 @@ def _generate_sample(experiment_run: ExperimentRun, sample: Sample, derived_samp
 
     row.lane = convert_alpha_digit_coord_to_ordinal(sample.coordinates, container_spec.coordinate_spec)
 
+    try:
+        experiment_run_process_measurement = ProcessMeasurement.objects.get(lineage__child_id=sample.pk)
+        try:
+            sequencer_position_property_value = PropertyValue.objects.get(object_id=experiment_run_process_measurement.pk, property_type__name="Sequencer Position")
+            row.sequencer_position = sequencer_position_property_value.value
+        except PropertyValue.DoesNotExist:
+            row.sequencer_position = None
+    except Exception as err:
+        row.sequencer_position = None
+
```

Comment:
Can't you just have one `except` clause with `except Exception as err`?

Reply:
I presume you don't expect any other model to have property type "Sequencer Position"?